Obsidian/Entities/Player.cs

```diff
@@ -321,7 +322,7 @@ public async Task RespawnAsync()
 
         this.visiblePlayers.Clear();
 
-        Registry.Dimensions.TryGetValue("minecraft:overworld", out var codec);
+        Registry.Dimensions.TryGetValue(0, out var codec);
```

Comment:
does 0 correlate to overworld? should this be an enum? (e.g. dimension)

Reply:
0 seems to be a magic number but I might be wrong ofc

Reply:
Yeah 0 is for overworld I forgot to change this to use `Registry.GetDimensionCodecOrDefault(string name)` will fix that in a bit ~